package com.tech.challenge.util.exception.vaga;

public class VagaAlreadyExistsException extends Exception{

    public VagaAlreadyExistsException(String menssage){super(menssage);}
}
